spec/classes/init_spec.rb

```diff
@@ -865,6 +719,18 @@
         :invalid => ['./invalid',3,2.42,['array'],a={'ha'=>'sh'}],
         :message => 'is not an absolute path',
       },
+      'regex_file_mode' => {
+        :name    => ['crontab_mode', 'cron_allow_mode', 'cron_deny_mode', 'cron_dir_mode'],
+        :valid   => ['0755','0750'],
+        :invalid => ['invalid','755',0755,'0980',2.42,['array'],a={'ha'=>'sh'}],
```

Comment:
@ghoneycutt this better check will make it into next release for sure! I have made a comment about that in the new spec [1]

Still working on refactoring the spec tests. Will not change functionality beforehand.

[1] https://github.com/Ericsson/puppet-module-cron/pull/26/files#diff-a446e4110d88d59f153d0721dad568feR690